eqcorrscan/utils/plotting.py

```diff
@@ -852,43 +853,42 @@ def detection_multiplot(stream, template, times, streamcolour='k',
-            try:
-                normalizer = max(image.data[int((template_times[0] -
-                                                image_times[0]).
-                                                total_seconds() /
-                                                image.stats.delta):
-                                            int((template_times[-1] -
-                                                 image_times[0]).
-                                                total_seconds() /
-                                                image.stats.delta)] /
-                                 max(image.data))
-            except ValueError:
-                # Occurs when there is no data in the image at this time...
-                normalizer = max(image.data)
-            normalizer /= max(template_tr.data)
-            axis.plot(template_times,
-                      template_tr.data * normalizer,
-                      templatecolour, linewidth=1.2)
+        time = times[i]
+        time_dt = time.datetime
+        template_times = [time_dt +
+                          dt.timedelta((j * template_tr.stats.delta) /
+                                       86400)
+                          for j in range(len(template_tr.data))]
+        # Normalize the template according to the data detected in
+        normalizer = max(image.data[int((template_times[0] -
+                                        image_times[0]).
+                                        total_seconds() /
+                                        image.stats.delta):
+                                    int((template_times[-1] -
+                                         image_times[0]).
+                                        total_seconds() /
+                                        image.stats.delta)] /
+                         max(image.data))
+        normalizer /= max(template_tr.data)
+        axis.plot(template_times,
+                  template_tr.data * normalizer,
+                  templatecolour, linewidth=1.2)
         ylab = '.'.join([template_tr.stats.station,
                          template_tr.stats.channel])
         axis.set_ylabel(ylab, rotation=0,
                         horizontalalignment='right')
-        # axis.yaxis.set_ticks([])
+        axis.yaxis.set_ticks([])
     if len(template) > 1:
-        axes[len(axes) - 1].set_xlabel('Time')
+        axes[-1].set_xlabel('Time (s)')
+        axes[-1].xaxis.set_major_locator(mdates.SecondLocator())
+        axes[-1].xaxis.set_major_formatter(mdates.DateFormatter('%H:%M:%S'))
+        xaxis.set_major_formatter(mdates.DateFormatter('%H:%M:%S'))
     else:
         axis.set_xlabel('Time')
+        axis.xaxis.set_major_locator(mdates.SecondLocator())
+        axis.xaxis.set_major_formatter(mdates.DateFormatter('%H:%M:%S'))
     plt.subplots_adjust(hspace=0, left=0.175, right=0.95, bottom=0.07)
-    plt.xticks(rotation=10)
+    # plt.xticks(rotation=10)
```

Comment:
I had the rotation there so plot-labels didn't overlap, is that not an issue now?